zebra-chain/src/value_balance.rs

```diff
@@ -165,17 +167,36 @@ impl ValueBalance<NonNegative> {
     /// `utxos` must contain the [`Utxo`]s of every input in this block,
     /// including UTXOs created by earlier transactions in this block.
     ///
-    /// "If any of the "Sprout chain value pool balance", "Sapling chain value pool balance",
-    /// or "Orchard chain value pool balance" would become negative in the block chain created
-    /// as a result of accepting a block, then all nodes MUST reject the block as invalid.
+    /// # Consensus
+    ///
+    /// > If the Sprout chain value pool balance would become negative in the block chain
+    /// > created as a result of accepting a block, then all nodes MUST reject the block as invalid.
+    ///
+    /// <https://zips.z.cash/protocol/protocol.pdf#joinsplitbalance>
+    ///
+    /// > If the Sapling chain value pool balance would become negative in the block chain
+    /// > created as a result of accepting a block, then all nodes MUST reject the block as invalid.
     ///
-    /// Nodes MAY relay transactions even if one or more of them cannot be mined due to the
-    /// aforementioned restriction."
+    /// <https://zips.z.cash/protocol/protocol.pdf#saplingbalance>
     ///
-    /// https://zips.z.cash/zip-0209#specification
+    /// > If the Orchard chain value pool balance would become negative in the block chain
+    /// > created as a result of accepting a block , then all nodes MUST reject the block as invalid.
+    ///
+    /// <https://zips.z.cash/protocol/protocol.pdf#orchardbalance>
     ///
     /// Zebra also checks that the transparent value pool is non-negative,
     /// which is a consensus rule derived from Bitcoin.
+    /// Individual transactions have a value pool that must be non-negative,
+    /// so the Transparent chain value pool balance (which is the sum of all the value pools for each
+    /// transaction and for each block in the chain) must be non-negative too.
+    /// > The remaining value in the transparent transaction value pool MUST be nonnegative.
+    ///
+    /// <https://zips.z.cash/protocol/protocol.pdf#transactions>
+    ///
+    /// > Nodes MAY relay transactions even if one or more of them cannot be mined due to the
+    /// > aforementioned restriction."
```

Comment:
There are some minor formatting typos here:
```suggestion
    /// transaction and for each block in the chain) must be non-negative too.
    ///
    /// > The remaining value in the transparent transaction value pool MUST be nonnegative.
    ///
    /// <https://zips.z.cash/protocol/protocol.pdf#transactions>
    ///
    /// > Nodes MAY relay transactions even if one or more of them cannot be mined due to the
    /// > aforementioned restriction.
```